migrations/202009100032.do.nys-exposure-thold10.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM settings WHERE settings_key = 'exposureConfig';
+INSERT INTO settings(settings_key, settings_value, is_language) VALUES('exposureConfig', '{"minimumRiskScore":1,"attenuationLevelValues":[2,3,4,5,6,7,8,8],"attenuationWeight":1,"daysSinceLastExposureLevelValues":[1,1,1,1,1,1,1,1],"daysSinceLastExposureWeight":1,"durationLevelValues":[1,1,1,1,1,1,1,1],"durationWeight":1,"transmissionRiskLevelValues":[1,1,1,1,1,1,1,1],"transmissionRiskWeight":1, "durationAtAttenuationThresholds": [56,62],"thresholdWeightings":[1,1,0],"timeThreshold":10}', false);
```

Comment:
Any reason a DELETE and INSERT is used instead of an UPDATE? There's a super small chance that the `exposureConfig` record is gone at the exact moment a DB read is executing. Or, if the INSERT fails then we have a longer period of time without a valid record.

Reply:
No reason. This should be in a transaction and that should address the state concern.